docs/configuration/configuration-file.mdx

```diff
@@ -5,224 +5,465 @@ import { StyledContent } from '../../components/styled.elements';
-- Reference uses it to apply your custom settings when building API reference documentation (hosted and on-premise).
-- VS Code extension uses it for linting criteria, to apply custom settings for live documentation preview, and to set API definition root files.
+* Workflows uses it in the API registry to manage your APIs and control advanced features like region and link resolution.
+* Reference uses it to apply your custom settings when it's building API reference documentation (hosted and on-premise).
+* VS Code extension uses it for linting criteria, to apply custom settings for live documentation previews, and to set API definition root files.
 
-## Filename and location
+## Location, name and format
 
-The Redocly configuration file must be named `redocly.yaml` or `.redocly.yaml`, and placed into your working directory or the root of your project (depending on the Redocly product you're using).
+The Redocly configuration file must be named either `redocly.yaml` or `.redocly.yaml`. It must be located in the root of your project directory, and it must be in the YAML format. If you cloned the `openapi-starter` project, this is already set up for you.
 
-OpenAPI CLI will search for the configuration file in the working directory.
+All of the available sections and config options are documented below.
 
-:::warning
+## `organization`
 
-Multiple configuration files in the same directory are not allowed.
-If you have a `redocly.yaml` and a `.redocly.yaml` file  in the same directory, Redocly apps will display a warning in the output and ask you to choose one file to use.
+Stores your Redocly Workflows organization ID that the `push` command uses to push API definition files to the API registry. If an organization ID is not passed explicitly in command-line arguments, `push` will look for it in `redocly.yaml`.
 
+:::info
+In Redocly Workflows, organization ID and organization name are not the same thing.
 :::
 
-## File format and structure
+To find the organization ID:
+
+1. Log into Workflows.
+2. Access the **API registry** page.
+3. In your browser's address bar, find the page URL.
+4. The segment after `app.redocly.com/org/` is the organization ID.
+
+For example, if the URL is `app.redocly.com/org/test_docs`, the organization ID is `test_docs`.
+
+## `apis`
+
+### Overview
 
-The Redocly configuration file must be in YAML format.
+The `apis` section lets you configure one or more OpenAPI definition files. This gives you the flexibility to reference specific files in commands, and configure each file at a granular level.
 
-The contents of the file should conform to the following structure:
+Every file listed in this section is identified by `name@version`. `@version` is optional. It allows you to derive multiple API definition versions from the same root file. When a version is not provided, Redocly apps interpret a file as being `latest`. Every `name@version` combination must be unique.
```

Comment:
suggestion: This doesn't seem right:

> It allows you to derive multiple API definition versions from the same root file.

Technically, it would work, but I'm not sure that's the most common way people use versioning (I could be wrong, though). Maybe you can just say "It allows you to manage multiple versions of each API." to catch all use-cases?